app/javascript/packages/document-capture/context/analytics.tsx

```diff
@@ -56,19 +57,27 @@ const AnalyticsContext = createContext<AnalyticsContextValue>({
 AnalyticsContext.displayName = 'AnalyticsContext';
 
 export function AnalyticsContextProvider({ children, trackEvent }: AnalyticsContextProviderProps) {
+  const { inPersonCtaVariantActive } = useContext(InPersonContext);
   const [submitEventMetadata, setSubmitEventMetadataState] = useState(DEFAULT_EVENT_METADATA);
   const setSubmitEventMetadata: SetSubmitEventMetadata = (metadata) =>
     setSubmitEventMetadataState((prevState) => ({ ...prevState, ...metadata }));
   const trackSubmitEvent: TrackSubmitEvent = (stepName) => {
     if (LOGGED_STEPS.includes(stepName)) {
+      if (stepName === 'location') {
+        submitEventMetadata.in_person_cta_variant = inPersonCtaVariantActive;
```

Comment:
does this need to go through a call to `setSubmitEventMetadataState` since it's modifying a shared object?

Reply:
Nailed it - I was wondering why I was struggling so darn bad with getting this to behave. I wish I'd seen this comment earlier!